src/main/assembly/dist/install/fileitems/solrconfig.xml

```diff
@@ -35,7 +35,7 @@
        that you fully re-index after changing this setting as it can
        affect both how text is indexed and queried.
   -->
-  <luceneMatchVersion>6.6.1</luceneMatchVersion>
+ <!-- <luceneMatchVersion>7.5.0</luceneMatchVersion>
```

Comment:
The lucene version is not being used in the configuration. I opened the comment to combine this with the next comment, but leave it available in case it needs to be used again later.